src/app/(nextjs_migration)/(authenticated)/user/breaches/page.tsx

```diff
@@ -157,6 +157,11 @@ export default async function UserBreaches() {
       <Script type="module" src="/nextjs_migration/client/js/breaches.js" />
       <Script type="module" src="/nextjs_migration/client/js/dialog.js" />
       <main data-partial="breaches">
+        <section>
+          {(process.env.PREMIUM_ENABLED === "true" && !session?.user.fxa?.subscriptions?.includes("monitor")) ?
+            <a className="button primary" href={process.env.SUBSCRIBE_PREMIUM_URL}>Subscribe to Premium</a>
```

Comment:
Q: should this be localized?

Reply:
This is just for our own debugging purposes, so I wouldn't bother our localisers with it.

Reply:
I think this feature is going to be English an US-only, at least at first, so we probably want to avoid bothering localizers.